x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/cleaner/AbstractIndicesCleanerTestCase.java

```diff
@@ -196,48 +195,48 @@ protected void createWatcherHistoryIndex(final DateTime creationDate) {
     /**
      * Creates a watcher history index from the specified version.
      */
-    protected void createWatcherHistoryIndex(final DateTime creationDate, final String version) {
-        final DateTimeFormatter formatter = DateTimeFormat.forPattern("YYYY.MM.dd").withZoneUTC();
-        final String index = ".watcher-history-" + version + "-" + formatter.print(creationDate.getMillis());
+    protected void createWatcherHistoryIndex(final ZonedDateTime creationDate, final String version) {
+        DateFormatter formatter = DateFormatter.forPattern("YYYY.MM.dd").withZone(ZoneOffset.UTC);
```

Comment:
`YYYY` in java time means week based year, where is in joda time it means year of era - this is lower case `yyyy` in java time

see https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html
see https://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html